src/pages/Events/AccessibilityReminders.tsx

```
@@ -0,0 +1,269 @@
+import React, { useState } from 'react';
```

ESLint

[error] Replace `'react'` with `"react"`

(prettier/prettier)